maxim/logger/groq/utils.py

```
@@ -163,64 +165,61 @@ def parse_chunks_to_response(content: str, usage_data: Optional[CompletionUsage]
         )
 
     @staticmethod
-    def parse_completion(completion: ChatCompletion) -> Dict[str, Any]:
-        """Parse Groq completion response into standardized format.
+    def parse_tool_calls(message: ChatCompletionMessage) -> Optional[List[ToolCallConfigDict]]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)